File: code-examples/java-rs/collation/find-sort-collation.java
// Create a client 
MongoClient mongoClient = MongoClients.create();

// Get the database and collection
MongoDatabase database = mongoClient.getDatabase("myDb");
MongoCollection<Document> collection = database.getCollection("myColl");

// Build the query filter
Bson filter = Filters.eq("city", "New York");

// Build the collation
Collation collation = Collation.builder().locale("de").build();

// Build the sort criteria
Bson sortCriteria = Sorts.ascending("name");

// Execute the query
FindPublisher<Document> results = collection.find(filter)
                                            .collation(collation)
                                            .sort(sortCriteria);

File: code-examples/scala/collation/find-sort-collation.scala
/*
   Retrieve documents that match "New York" in the "city" field,
   sorted by the "name" field by using the German collation.
*/
// start find sort collation
val collation = Collation.builder().locale("de").build()
collection.find(Filters.eq("city", "New York")).sort(Sorts.ascending("name")).collation(collation).iterator()
// end find sort collation
